account.go

```diff
@@ -330,9 +333,12 @@ type AccountBusinessProfile struct {
 
 // AccountCapabilities is the resource representing the capabilities enabled on that account.
 type AccountCapabilities struct {
-	CardPayments   AccountCapabilityStatus `json:"card_payments"`
-	LegacyPayments AccountCapabilityStatus `json:"legacy_payments"`
-	Transfers      AccountCapabilityStatus `json:"transfers"`
+	AUBECSDebitPayments    AccountCapabilityStatus `json:"au_becs_debit_payments"`
+	CardPayments           AccountCapabilityStatus `json:"card_payments"`
+	LegacyPayments         AccountCapabilityStatus `json:"legacy_payments"`
+	TaxReportingUS1099K    AccountCapabilityStatus `json:"tax_reporting_us_1099_k"`
+	TaxReportingUS1099MISC AccountCapabilityStatus `json:"tax_reporting_us_1099_misc"`
```

Comment:
```suggestion
	TaxReportingUS1099Misc AccountCapabilityStatus `json:"tax_reporting_us_1099_misc"`
```

Reply:
Why? Isn't MISC an acronym?

Reply:
I think it's just an abbreviation for "miscellaneous"?

Reply:
Hm, but the 1099 form is called "Form 1099-MISC". So your version is probably fine.